### PR TITLE
HTML 27 create component forms

### PR DESCRIPTION
**Pull request by cajimenez96**

### CODE REVIEW

File: src/components/FormsContact/FormsContact.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          <input type="text" className="form-control" id="fullName" placeholder="Ingrese su nombre completo" />
+        </div>
+        <div className="mb-3">
+          <label htmlFor="email" className="form-label">Correo Electrónico</label>
+          <input type="email" className="form-control" id="email" placeholder="Ingrese su correo electrónico" />
+        </div>
+        <div className="row justify-content-between">
+          <div className="mb-3">
+            <label htmlFor="tel" className="form-label">Teléfono</label>
+            <input type="tel" className="form-control" id="tel" placeholder="Ingrese su número de teléfono" />
```
Comment (patoogm) [inline src/components/FormsContact/FormsContact.js:19]: type='tel' not exist. This correct is type=number.

**Review by patoogm — CHANGES_REQUESTED** [review]

File: src/components/FormsContact/FormsContact.js (hunk 1 above)
Comment (cajimenez96) [reply src/components/FormsContact/FormsContact.js]: ok

File: src/components/FormsContact/FormsContact.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          <input type="text" className="form-control" id="fullName" placeholder="Ingrese su nombre completo" />
+        </div>
+        <div className="mb-3">
+          <label htmlFor="email" className="form-label">Correo Electrónico</label>
+          <input type="email" className="form-control" id="email" placeholder="Ingrese su correo electrónico" />
+        </div>
+        <div className="row justify-content-between">
+          <div className="mb-3">
+            <label htmlFor="tel" className="form-label">Teléfono</label>
+            <input type="text" className="form-control" id="tel" placeholder="Ingrese su número de teléfono" />
```
Comment (patoogm) [inline src/components/FormsContact/FormsContact.js:19]: This is incorrect. Phone number is not a text, is a number.

File: src/components/FormsContact/FormsContact.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          </div>
+          <div className="mb-3">
+            <label htmlFor="time" className="form-label">Horario de contacto</label>
+            <input type="text" className="form-control" id="time" value="Lun. a Vie. 9hs-18hs y Sab. 9hs-12hs" disabled />
```
Comment (patoogm) [inline src/components/FormsContact/FormsContact.js:23]: This type text is incorrect. type="date" is correct

**Review by patoogm — CHANGES_REQUESTED** [review]

**Review by patoogm — APPROVED** [review]

**Pull request merged**